.github/workflows/test_devices_plugin.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Lint with PyLint
         shell: bash -el {0}
         run: |
-          pylint plugins/devices/src --rcfile .pylintrc --disable E0401,E0203
+          pylint plugins/devices/src --rcfile .pylintrc --disable E0401,E0203,C0411
```

Comment:
Import order does not (or at least should not) matter. It is just a coding standard.
I guess we just got tired of it?